src/resources/texture.js

```diff
@@ -136,6 +136,13 @@ Object.assign(pc, function () {
         texture._levelsUpdated = texture._cubemap ? [[true, true, true, true, true, true]] : [true];
     };
 
+    /**
+     * @class
+     * @name pc.TextureHandlerOptions
+     * @description The supported {@link pc.Asset} options when loading texture assets.
+     * @property {boolean} [crossOrigin] - For browser-supported image formats only, enable cross origin.
+     */
```

Comment:
Where's the constructor? Have you tried compiling Typescript that uses this? I'm guessing the compiler will complain that a vanilla object can't be cast to a `pc.TextureHandlerOptions`.

Reply:
removed.